src/main/java/gov/nasa/pds/tools/label/LocationValidator.java

```diff
@@ -226,8 +226,10 @@ public void validate(ValidateProblemHandler problemHandler, URL url) {
 			ruleContext.setCrawler(crawler);
 			ruleContext.setRule(rule);
 
+            LOG.info("validate:Submitting task {} to taskManager location {} rule {} ",task,location, rule.getCaption());
```

Comment:
@qchaupds can we make this a DEBUG message? we don't want to overload the INFO with even more information that is more for operational use.

Reply:
The info logging has been removed for next commit.